src/utils/conversion.jl

```diff
@@ -0,0 +1,7 @@
+"Convert Tuple to Min Max Named Tuple"
+Base.convert(::Type{Min_Max}, input::Tuple{Float64, Float64}) =
+    (min = input[1], max = input[2])
+
+"Convert Tuple to Up Down Named Tuple"
+Base.convert(::Type{Up_Down}, input::Tuple{Float64, Float64}) =
+    (up = input[1], down = input[2])
```

Comment:
Are these necessary?  I don't see a call site. You can do this instead:
```
a = (3.4, 5.6)
Min_Max(a)
(min = 3.4, max = 5.6)
```

Reply:
The code is used in the parsing of the dynamics. These are defined and automatically used by Julia when this happens

```Julia
struct foo
    a::Min_Max
end

foo((1.0, 2.0))
```